src/net.cpp

```diff
@@ -642,6 +647,7 @@ bool CNode::ReceiveMsgBytes(const char *pch, unsigned int nBytes, bool& complete
 {
     complete = false;
     int64_t nTimeMicros = GetTimeMicros();
+    LOCK(cs_vRecv);
```

Comment:
Just trying to use a big hammer here for 0.14. The lock only contends with copyStats.